app/page/template/modal/group-creation.htm

```diff
@@ -36,7 +36,7 @@
                data-uie-name="enter-group-name">
         <div class="group-creation-error" data-bind="text: nameError, visible: nameError().length" data-uie-name="error-group-name"></div>
         <!-- ko if: isTeam -->
-          <div class="group-size-info" data-bind="l10n_text: {id: z.string.groupSizeInfo, substitute: z.conversation.ConversationRepository.CONFIG.GROUP.MAX_SIZE}"></div>
+          <div class="group-size-info" data-bind="l10n_text: {id: z.string.groupSizeInfo, substitute: z.conversation.ConversationRepository.CONFIG.GROUP.MAX_SIZE}" data-uie-name="status-group-size-info"></div>
```

Comment:
You can also assign the `MAX_SIZE` to a [data-uie-status](https://github.com/wireapp/wire-webapp/wiki/data-attributes) property.